src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -259,6 +259,7 @@ private void LogMessage(TestMessageLevel testMessageLevel, string message)
         private void InitializeExtensions(IEnumerable<string> sources)
         {
             var extensions = TestPluginCache.Instance.GetExtensionPaths(TestPlatformConstants.TestAdapterEndsWithPattern, this.skipDefaultAdapters);
```

Comment:
>GetExtensionPaths [](start = 54, length = 17)

Can we pass the datacollectorendswithpattern to this function ?

Reply:
GetExtensionPaths takes only one endsWithPattern at a time.